homework/src/main/java/io/codelex/kontroldarbs/exercise2/Basket.java

```diff
@@ -0,0 +1,57 @@
+package io.codelex.kontroldarbs.exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+
+    private List<T> list;
+    private Integer itemCount;
+
+    public Basket() {
+        this.list = new ArrayList<>();
+        this.itemCount = 0;
+    }
+
+    public List<T> getList() {
+        return list;
+    }
+
+    public void setList(List<T> list) {
```

Comment:
Ar šo var Tavu klasi salauzt - ja mēs uzstādīsim iekšējo sarakstu ar setList(), tad itemCount neatbildīs tam, kas faktiski ir iekš list.

Reply:
Jā, šeit es kaut kā ātrumā uzģenerēju getters/setters un tad nebija pārāk daudz laika domāt par to, ko no tā visa vajag/nevajag. Novācu šo setter.